ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -175,29 +216,37 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
     );
   }
 
-  private handleGoBack = () =>
-    Alert.alert(
-      I18n.t("onboarding.alert.title"),
-      I18n.t("onboarding.alert.description"),
-      [
-        {
-          text: I18n.t("global.buttons.cancel"),
-          style: "cancel"
-        },
-        {
-          text: I18n.t("global.buttons.exit"),
-          style: "default",
-          onPress: () => this.props.dispatch(abortOnboarding())
-        }
-      ]
-    );
+  private handleGoBack = () => {
+    if (this.isEditing) {
+      this.props.navigation.goBack();
+    } else {
+      Alert.alert(
+        I18n.t("onboarding.alert.title"),
+        I18n.t("onboarding.alert.description"),
+        [
+          {
+            text: I18n.t("global.buttons.cancel"),
+            style: "cancel"
+          },
+          {
+            text: I18n.t("global.buttons.exit"),
+            style: "default",
+            onPress: () => this.props.dispatch(abortOnboarding())
+          }
+        ]
+      );
+    }
+  };
 }
 
 function mapStateToProps(state: GlobalState) {
-  const optionProfile = pot.toOption(state.profile);
-
+  const optionProfile = pot.toOption(profileSelector(state));
+  // TODO: get info on validation from profile
+  //      https://www.pivotaltracker.com/story/show/168662501
+  const isEmailValidated = true;
```

Comment:
I don't know if MatteoDS changes his mind about it, but in case when the email is not validate don't we show this screen?
> Modale che si apre nel caso l'indirizzo email inserito non sia validato https://invis.io/VTTQ50YJAH4
It includes a different message and an image in addition

Reply:
The user can change his email at any time from the profile section:
- if email is validated, the user can access the EmailInsertScreen from the `EmailReadScreen` (orange arrow)
- if email is not validated, the user can modify his/her email from the [linked](https://invis.io/VTTQ50YJAH4) modal (pink arrow)
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/38431762/67667200-98b10400-f96d-11e9-966a-8e4d8708c774.png">

The isEmailValidated parameter is used to update the content, afdding or not the [and validated] item to the text up on the input label:
<img width="328" alt="image" src="https://user-images.githubusercontent.com/38431762/67667513-47554480-f96e-11e9-9fdf-907ced418aa5.png">

Within https://github.com/teamdigitale/io-app/pull/1363/commits/9862dde68e53271db2d13fc7170f1d99a4a37164 , TODOs are inserted to better trace future implementations needed

Reply:
@valentinacala thank you now I got it